test/support/factory.ex

```diff
@@ -1,7 +1,7 @@
 defmodule Wanda.Factory do
   @moduledoc false
 
-  use ExMachina
+  use ExMachina.Ecto, repo: Wanda.Repo
```

Comment:
What is this `Wanda.Repo` here for? Is it needed because the struct is databas schema?

Reply:
it allows to do `insert(:something)` and indeed have stuff in the db as I understood reading [this](https://hexdocs.pm/ex_machina/readme.html#overview)

Reply:
Ok, I see. Well, we don't use it yet, but we will, so feel free to leave it.

Reply:
Edit: I see fome `insert_list`, so yet, we use it hehe

Reply:
we use it here
https://github.com/trento-project/wanda/pull/36/files#diff-b5a8fea17b9ce629cd14125b9a03a7c9a81f206ce6b0b2d135624b1baa28cc0aR41